pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dask-image"
+authors = [{name="dask-image contributors"}]
+description = "Distributed image processing"
+keywords = ["dask-image", "dask", "image"]
+readme = "README.md"
+license = { file="LICENSE.txt" }
+dynamic = ["version"]
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "dask[array] >=2021.10.0",
+    "numpy >=1.11.3",
+    "scipy >=0.19.1",
+    "pims >=0.4.1",
+    "tifffile >=2018.10.18",
+]
+
+[project.optional-dependencies]
+test = [
+    "flake8 >=3.4.1",
+    "Flake8-pyproject",
```

Comment:
Maybe [FlakeHell](https://flakehell.readthedocs.io/) is a better option? This blogpost seems encouraging: https://dev.to/bowmanjd/using-flake8-and-pyproject-toml-with-flakehell-1cn1

Reply:
Moving this discussion to https://github.com/dask/dask-image/issues/307